src/include/threadpool/mono_queue_pool.h

```diff
@@ -4,32 +4,29 @@
 //
 // mono_queue_pool.h
 //
-// Identification: test/threadpool/mono_queue_pool.h
+// Identification: src/include/threadpool/mono_queue_pool.h
 //
 // Copyright (c) 2015-17, Carnegie Mellon University Database Group
 //
 //===----------------------------------------------------------------------===//
 
 #pragma once
 
+#include "settings/settings_manager.h"
 #include "worker_pool.h"
 
 namespace peloton {
 namespace threadpool {
 
-// TODO: tune these variables
-constexpr static size_t kDefaultTaskQueueSize = 32;
-constexpr static size_t kDefaultWorkerPoolSize = 4;
-
 /**
  * @brief Wrapper class for single queue and single pool.
  * One should use this if possible.
  */
 class MonoQueuePool {
  public:
-  MonoQueuePool()
-      : task_queue_(kDefaultTaskQueueSize),
-        worker_pool_(kDefaultWorkerPoolSize, &task_queue_),
+  MonoQueuePool(int task_queue_size = 32, int worker_pool_size = 4)
```

Comment:
I don't think we want the default numbers here. I think we should explicitly remove the default constructor (`MonoQueuePool() = delete;`), and remove the default values here. People should know what they're doing and give explicit values to the QueuePool.